.gitignore

```diff
@@ -113,7 +113,6 @@ celerybeat.pid
 *.sage.py
 
 # Environments
-.env
```

Comment:
was this intended?

Reply:
Yes, the devcontainer extension uses a .env file for configuration. This file would need to be checked in to the repo. Our setup doesn't use that file right now, but we should alter the gitignore so there are no surprises for future users.

https://stackoverflow.com/a/58227846